tests/assert/assert.go

```diff
@@ -21,3 +21,13 @@ func Error(t *testing.T, err error, expected string) {
 		t.Fatalf("expected '%s' but got '%s'", expected, err.Error())
 	}
 }
+
+// True asserts that an interface is equal to boolean "true"
+func True(t *testing.T, expected interface{}) {
```

Comment:
Can we match the signature of [testify's `assert.True`](https://godoc.org/github.com/stretchr/testify/assert#True) so that we can drop-in-replace it later?